app/models/validations/setup_validations.rb

```diff
@@ -90,6 +90,10 @@ def validate_scheme_has_confirmed_locations_validation(record)
   end
 
   def validate_scheme(record)
+    if record.scheme&.status == :incomplete
+      record.errors.add :scheme_id, :incomplete, message: I18n.t("validations.setup.scheme.incomplete")
+    end
+
```

Comment:
Would this invalidate those logs that have incomplete scheme assigned to them for some time?
I assume we're not showing incomplete schemes in the scheme dropdown and this happened by setting the schemes somewhere else? Any idea where?

Reply:
It will invalidate logs with an incomplete scheme assigned to them - I'm waiting for an answer from Rachel around giving users the opportunity to just complete the scheme first (rather than clearing data) before I write anything to deal with that.

Logs could have got to that state via bulk upload, or possibly import from old core.

I think we don't show schemes where the scheme data isn't complete for the scheme id question, but they can also be incomplete if they have no complete locations and I think in that case we would show them (the filter is based on the 'confirmed' property) - this seems fine in terms of the user seeing why they can't select a particular thing, so I left it alone.